### PR TITLE
Fix UI issues found by QA

### PR DESCRIPTION
**Pull request by fffw**

### CODE REVIEW

File: src/github.com/getlantern/lantern-ui/app/partials/feed.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <a class="item" target="_blank" href="{{feed.link}}" ng-click="trackFeed(feedsTitle)">
   <div ng-if="feed.image" class="feed-thumbnail">
-    <img ng-src="{{feed.image}}" on-error="hideImage(feed)"/>
+    <img ng-src="{{feed.image}}" on-error="hideImage(feed)" data-pin-no-hover="true" />
```
Comment (fffw) [inline src/github.com/getlantern/lantern-ui/app/partials/feed.html]: Fix #4007

File: src/github.com/getlantern/lantern-ui/app/_css/news.css [hunk 2]
```diff
@@ -179,6 +179,9 @@
 #news h4 {
   font-weight: 700;
   line-height: 18px;
+  white-space: nowrap;
```
Comment (fffw) [inline src/github.com/getlantern/lantern-ui/app/_css/news.css]: Fix #4006 by showing single line

File: src/github.com/getlantern/lantern-ui/app/_css/news.css [hunk 3]
```diff
@@ -208,6 +211,10 @@
   background: #fafbfb;
 }
 
+#news p.snippet img {
+  display: none;
```
Comment (fffw) [inline src/github.com/getlantern/lantern-ui/app/_css/news.css]: Fix #3997

**Comment by myleshorton** [comment]
LGTM @fffw!

**Pull request merged**